doc/stdlib/strings.rst

```diff
@@ -333,7 +333,7 @@
 
    ``strings`` can be any iterable over elements ``x`` which are convertible to strings via ``print(io::IOBuffer, x)``\ .
 
-.. function:: chop(string)
+.. function:: chop(string, i = 1)
 
    .. Docstring generated from Julia source
```

Comment:
if you don't have a source build, you can just make the change below. though rst has different formatting than markdown so the `i` will need double backticks in the rst file